Project.toml

```diff
@@ -28,6 +29,7 @@ PNGFiles = "0.3, 0.4"
 QOI = "1"
 Sixel = "0.1.2"
 TiffImages = "0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 0.10"
+WebP = "0.1.2"
```

Comment:
```suggestion
WebP = "0.1"
```

Reply:
The restriction to >= 0.1.2 was on purpose - there were some bad bugs in 0.1.0 and 0.1.1.

Reply:
Shouldn't it be `>=0.1.2` then ? I always forget if the `>=` is implied or if it pins the version.

Reply:
`0.1.2` means `>=0.1.2, <0.2.0` which is semantically differnt from `>=0.1.2`.

Reply:
Always forget this one, thanks for the correction 😌.